DFC.ServiceTaxonomy.JobProfiles.Module/Migrations.cs

```diff
@@ -71,5 +71,27 @@ public int Create()
 
             return 1;
         }
+
+        public int UpdateFrom1()
+        {
+            _contentDefinitionManager.AlterPartDefinition(nameof(JobProfile), part => part
+                .Attachable()
+                .WithDescription("Creates JobProfile related metadata record.")
+            );
+
+            SchemaBuilder.AlterIndexTable<JobProfileIndex>(table => table.AddColumn<string>(nameof(JobProfileIndex.JobProfileTitle)));
```

Comment:
Can this column be added to Create method instead as this code hasn't been deployed anywhere yet, so a single create method should be okay instead of a create and an update...? Could be an issue for you locally though.